manifests/post-installation/flannel-template.yaml

```
@@ -20,3 +20,7 @@ spec:
           skipCNIConfigInstallation: false
         imagePullSecrets:
           - name: dpf-pull-secret
+  resourceRequirements:
+    cpu: 1
+    memory: 1Gi
+    storage: 1Gi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)